FantasmoSDK/Classes/FMLocationManager.swift

```diff
@@ -72,9 +72,12 @@ open class FMLocationManager: NSObject, FMApiDelegate {
     private var lastFrame: ARFrame?
     private var lastLocation: CLLocation?
     
-    private weak var delegate: FMLocationDelegate?
+    private var delegate: FMLocationDelegate?
```

Comment:
This should be `weak` no?

Reply:
Yes, this is my fail when merging changes from "develop" branch. Thanks!